src/context/UserContext.jsx

```diff
@@ -4,10 +4,14 @@ import { getUser } from '../services/users';
 const UserContext = createContext();
 
 const UserProvider = ({ children }) => {
+  // Is this just for initializing? 
```

Comment:
just for initializing _if_ there is a user logged into the current session (sort of like when we stored a user in localStorage). already logged in -> current user is set in state; not logged in -> user state is initialized as `{}`